virttest/utils_libvirtd.py

```diff
@@ -191,18 +191,22 @@ class LibvirtdSession(object):
     def __init__(self, gdb=False,
                  logging_handler=None,
                  logging_params=(),
-                 logging_pattern=r'.*'):
+                 logging_pattern=r'.*',
+                 service_name=None):
         """
         :param gdb: Whether call the session with gdb debugging support
         :param logging_handler: Callback function to handle logging
         :param logging_pattern: Regex for filtering specific log lines
+        :param service_name: Service name such as virtqemud or libvirtd
         """
         self.gdb = None
         self.tail = None
         self.running = False
         self.pid = None
+        self.service_name = service_name
         self.bundle = {"stop-info": None}
-        self.libvirtd_service = Libvirtd()
+        self.libvirtd_service = Libvirtd(service_name=self.service_name)
+        self.service_exec = self.libvirtd_service.service_list[0]
```

Comment:
Better add comment to explain why self.service_exec  is assigned to .libvirtd_service.service_list[0], rather than other element in the list

Reply:
done.